### PR TITLE
Jakec/update push notification text

### PR DESCRIPTION
**Pull request by JakeCooper**

### CODE REVIEW

File: app/src/main/java/antidose/antidose/AntidoseNotifications.java [hunk 1]
```diff
@@ -121,9 +138,9 @@ public void createNotification(Location location, int max, String incidentId){
                         .setDeleteIntent(createOnDismissedIntent(this, incidentId))
                         .setAutoCancel(true)
                         .setSmallIcon(R.mipmap.ic_launcher)
-                        .setContentTitle("Someone is experiencing an overdose " +
-                                distance + "m away")
-                        .setContentText("Click to respond");
+                        .setContentTitle("Overdose " +
+                                getNotificationDistance(distance) + " away!")
```
Comment (JordanVlieg) [inline app/src/main/java/antidose/antidose/AntidoseNotifications.java]: Could we maybe format this as "Please help, overdose 100m away."

**Review by geofflorne — APPROVED** [review]
looks good!

File: app/src/main/java/antidose/antidose/AntidoseNotifications.java [hunk 2]
```diff
@@ -87,15 +88,31 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
...
 
             // This is a notification dismissal
         } else if(notification.equals("dismiss")){
             cancelNotification();
         }
     }
 
+    public String getNotificationDistance(Float distance) {
+        String responseText = "";
+        if (distance > 1000.0) {
```
Comment (JordanVlieg) [inline app/src/main/java/antidose/antidose/AntidoseNotifications.java]: Should be >=

**Review by JordanVlieg — COMMENTED** [review]
fix ur shit

**Pull request merged**